test/unit/cli/test_utils_v2.py

```diff
@@ -0,0 +1,59 @@
+import json
+from typing import Optional
+
+import pytest
+from pydantic import Secret, ValidationError
+
+from unstructured_ingest.v2.cli.utils.click import extract_config
+from unstructured_ingest.v2.interfaces import AccessConfig, ConnectionConfig
+
+
+def test_extract_config_optional_access_config():
+    class MyAccessConfig(AccessConfig):
+        host: str
+
+    class MyModel(ConnectionConfig):
+        v: int
+        access_config: Optional[Secret[MyAccessConfig]]
```

Comment:
I was expecting the Optional to be in line 13 if we had multiple AccessConfigs and some are Optional.

Reply:
There should only ever be one access config

Reply:
We already have plenty of access configs that have optional fields and that works fine.

Reply:
I meant multiple optional fields in the AccessConfigs.  But sounds like it works fine already.

Reply:
Example of one that already exists that works: [milvus.py](https://github.com/Unstructured-IO/unstructured-ingest/blob/main/unstructured_ingest/v2/processes/connectors/milvus.py#L34)